tensorflow/g3doc/how_tos/adding_an_op/index.md

```diff
@@ -107,7 +107,7 @@ REGISTER_KERNEL_BUILDER(Name("ZeroOut").Device(DEVICE_CPU), ZeroOutOp);
 ```
 
 Once you
-[build and reinstall TensorFlow](../../get_started/os_setup.md#pip-installation), the
+[build and reinstall TensorFlow](../../get_started/os_setup.md#create-the-pip-package-and-install), the
```

Comment:
Before making any of the {#anchor} changes on #660, this link originally pointed to [this part of the file](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/g3doc/get_started/os_setup.md#pip-installation) (which #660 maintained, using the GitHub style anchor tag), as opposed to this change which points to [this part of the file](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/g3doc/get_started/os_setup.md#create-the-pip-package-and-install) several sections down. However, it looks like your change is probably going to a more relevant part of the page, as they will need to install from sources as opposed to 'easy-installing' from pip. 

I'm thinking maybe this line should read

```
[build and reinstall TensorFlow](../../get_started/os_setup.md#installing-from-sources), the
```

to point to the [very start of the "Installing from Sources"](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/g3doc/get_started/os_setup.md#installing-from-sources) section as opposed to the very end of it.

Thoughts? Probably over thinking a one-line change to a doc file, but this piqued my curiosity.